packages/graphql-types-generator/package.json

```diff
@@ -74,8 +74,7 @@
     ],
     "testMatch": [
       "**/test/**/*.(js|ts)",
-      "**/test/*.(js|ts)",
-      "**/__tests__/*.(js|ts)"
+      "**/test/*.(js|ts)"
```

Comment:
Since this is a regex matching, make sure we are not accidentally excluding tests other than that corresponding to `flow-modern` that is currently not supported in `amplify add codegen`.

Reply:
The typegen package tests are all in `test/` folder. Only the flow modern ones are in `__tests__/`